lib/event.rb

```diff
@@ -0,0 +1,27 @@
+
+class Event
+  attr_reader :ages, :name
+  def initialize(name, ages)
+    @ages = ages
+    @name = name
+  end
+
+  def max_age
+    ages.max
+  end
+
+  def min_age
+    ages.min
+  end
+
+  def average_age
+    ages_sum = ages.sum.round(1)
+    mean_age = (ages_sum / ages.length)
+  end
+
+  def standard_deviation
+    age_deviation = ages.map { |age| age - average_age }
+    squared = age_deviation.map { |deviation| deviation ** 2 }
```

Comment:
I would prefer these be broken out into helper methods. I would say this method is doing to much, and thus isn't following SRP (single responsibility principle)